apps/web/src/server/app/validation/validation.controller.js

```diff
@@ -328,7 +328,7 @@ export function getCheckAndConfirm(request, response) {
 	let backURL;
 
 	if ('invalidAppealDetails' in appealWork) {
-		backURL = `/validation/${routes.invalidAppealOutcome.path}?direction=back`;
+		backURL = `/validation/${routes.reviewAppealRoute.path}/${appealData.AppealId}?direction=back`;
```

Comment:
I don't think this change is correct. The back link _should_ take you to the invalid appeal outcome page.

The issue here is that the `changeOutcomeURL` is using whatever the back link is using, which is not correct, as the two destinations can differ.

Reply:
Yes you're absolutely right. On closer review of the ACs these need to go to different destinations, which makes sense given they are separate links! This is fixed now.